tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -3800,10 +3792,6 @@ TEST_F(EnergyPlusFixture, OutputReportTabular_GatherHeatEmissionReport)
     EXPECT_EQ(2 * reliefEnergy * DataGlobals::convertJtoGJ, BuildingPreDefRep.emiHVACRelief);
     EXPECT_EQ(condenserReject + coilReject, DataHeatBalance::SysTotalHVACRejectHeatLoss);
     EXPECT_EQ(2 * condenserReject * DataGlobals::convertJtoGJ + coilReject * DataGlobals::convertJtoGJ, BuildingPreDefRep.emiHVACReject);
-
```

Comment:
No need to call ```clear_state()``` explicitly after unit tests. This happens automatically by the test fixture before each test.

Reply:
:+1: :+1: